notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    import gradio as gr
</pre><p>user can have not installed gradio as it is not listed in our requirements</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>